OpenEXR/IlmImfTest/testBadTypeAttributes.cpp

```diff
@@ -127,6 +127,7 @@ template<class T> void readTiledThing(T & input,bool test)
 		case NUM_LEVELMODES:
 		default:
 			std::cerr << "Invalid tile mode " << int(t.mode) << std::endl;
+            x_levels = y_levels = 0;
```

Comment:
indentation seems off (tab vs. space?)

Reply:
oh, the whole repository is full of this, where the previous style was a mix of sometimes all spaces, sometimes tabs + spaces. I am hoping to run a clang-format over the whole repository as soon as possible

Reply:
fixed the surrounding regions by converting the local region to spaces to match the rest of the file.